msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -205,10 +205,6 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 		Inputs="@(_CompileToNativeInput);@(_FileNativeReference);@(BundleDependentFiles)"
 		Outputs="$(_AppBundlePath)Contents\MacOS\$(_AppBundleName);$(DeviceSpecificOutputPath)bundler.stamp">
 
-		<ItemGroup>
-			<_FrameworkNativeReference Include="@(_FrameworkNativeReference -> '%(Identity)/%(Filename)')" Condition="'%(Extension)' == '.framework' Or '%(Extension)' == '.xcframework'" />
```

Comment:
Should the identical code in Xamarin.iOS.Common.targets be removed too?

Reply:
definitively!